.github/workflows/check-configs-changes.yml

```diff
@@ -0,0 +1,70 @@
+name: "Check Config Changes"
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, labeled, unlabeled]
```

Comment:
Labeled and unlabeled were needed on the previous approach but not here. Let's remove them. And since opened, synchronize and reopened are the default types, I think we should omit it entirely.